### DISCUSSION TITLE
Can `refreshInterval` be added to wagmi queries?

### DISCUSSION THREAD

**Discussion by kyscott18** [wevm/wagmi#2230] [Ideas]
Tanstack `react-query` has a `refetch-interval` parameter that is useful for refetching data, especially on networks where refetching on every block is too expensive, such as arbitrum. 

I am willing to make the pr, just making sure there isn't a reason that this isn't being done already.

**Comment by tmm** [comment]
Going to expose all the options as part of `wagmi@1`. More info here: https://github.com/wagmi-dev/wagmi/discussions/2207

**Reply by kyscott18** [reply]
wagmi@1 or wagmi@2?

**Reply by jxom** [reply]
wagmi v2 :)
